packages/tx-construction/src/tx-builder/types.ts

```diff
@@ -139,7 +139,7 @@ export interface SignedTx {
  */
 export interface UnsignedTx {
   inspect(): Promise<TxInspection>;
-  sign(): Promise<SignedTx>;
```

Comment:
Let's also delete the type that's no longer used.

Reply:
Is this no longer used? I think it is, but we now pass the `SubmitTxArgs` to the sign method as opposed to the `SignedArgs` to it.  So all the signedArgs types have been removed, and we now pass different ones. We could rename the method from sign to submit, but TxBuilder still uses a sign, and I think we need a sign (or submit) to get a txOut.

Reply:
I think we no longer use https://github.com/input-output-hk/cardano-js-sdk/pull/751/files#diff-d2eebb6c7502396b4d246766bf4f31551ecdef4f174f4be4f9784c69bed987e3R126 type. If that's true, then there's no reason to keep it.

Reply:
Ah yes. I'll remove the interface.

Reply:
2d88f9ab41c0bf941b8126409e8d597fa2713303  should address this